examples/opentelemetry-example-app/tests/test_flask_example.py

```diff
@@ -15,11 +15,11 @@
 import unittest
 from unittest import mock
 
+import opentelemetry_example_app.flask_example as flask_example
```

Comment:
pylint is suggesting to undo this change: https://travis-ci.org/open-telemetry/opentelemetry-python/jobs/646105484#L2212